### PR TITLE
Opendsa small fixes

### PR DESCRIPTION
**Pull request by mint87**
https://codio.myjetbrains.com/youtrack/issue/codio-11916

### CODE REVIEW

File: converter/rst2markdown.py [hunk 1]
```diff
@@ -245,15 +244,14 @@ def _code_lines(self, data):
         return "\n".join(lines)
 
     def _extrtoolembed(self, matchobj):
-        caret_token = self._caret_token
         name = matchobj.group('name').lower()
         ex_data = self._exercises.get(name.lower(), {})
         assessment_id = f'test-{name.lower()}'
         if not ex_data:
-            return f'{caret_token}***Exercise:*** "{name}" **not found**{caret_token}{caret_token}'
+            return
```
Comment (destitutus) [inline converter/rst2markdown.py:251]: should it return an empty string?

Comment (mint87) [reply converter/rst2markdown.py:251]: Yes. To avoid displaying exercise titles on the page where they are missing

Comment (destitutus) [reply converter/rst2markdown.py:251]: What is the difference with line 254? Not sure if I understand it

Comment (mint87) [reply converter/rst2markdown.py]: fixed it

**Review by destitutus — APPROVED** [review]

**Pull request merged**